src/main/webapp/components/BaseFormView.jsx

```diff
@@ -1026,8 +1033,7 @@ class BaseFormView extends PureComponent {
                         try {
                             this.hook.onRender();
                         } catch (err) {
-                            // eslint-disable-next-line no-console
-                            console.error(err);
+                            onCustomHookError({ methodName: 'onRender' });
```

Comment:
I do not like this approach, it makes end user furious because he cannot say what is actually wrong. I'm fine with blocking the stack trace, but please log `err.name` and `err.message`
@artemrys ?

Reply:
I agree with Włodek, let's show as much as we can while hiding the sensitive information.

Reply:
I also don't like this approach. We should configure logging system and integrate it with Splunk RUM.